build/AzurePipelinesTemplates/ProjectReunion-RunHelixTests-Job.yml

```diff
@@ -32,99 +22,173 @@ jobs:
     maxParallel: ${{ parameters.maxParallel }}
     matrix: ${{ parameters.matrix }}
   variables:
-    artifactsDir: $(Build.SourcesDirectory)\Artifacts
-    taefPath: $(Build.SourcesDirectory)\build\Helix\packages\taef.redist.wlk.10.31.180822002\build\Binaries\$(buildPlatform)
-    helixCommonArgs: '/binaryLogger:$(Build.SourcesDirectory)/${{parameters.name}}.$(buildPlatform).$(buildConfiguration).binlog /p:HelixBuild=$(Build.BuildId).$(buildPlatform).$(buildConfiguration) /p:Platform=$(buildPlatform) /p:Configuration=$(buildConfiguration) /p:HelixType=${{parameters.helixType}} /p:TestSuite=${{parameters.testSuite}} /p:ProjFilesPath=$(Build.ArtifactStagingDirectory) /p:rerunPassesRequiredToAvoidFailure=${{parameters.rerunPassesRequiredToAvoidFailure}}'
+    # This should be kept in sync with the value in these files:
+    #   build\Helix\packages.config
+    #   test\ProjectReunion.Helix.TestCommon\ProjectReunion.Helix.TestCommon.csproj
+    winUIHelixVer: 0.5.2
+    winUIHelixPipelineScripts: build\Helix\packages\Microsoft.Internal.WinUI.Helix.$(winUIHelixVer)\scripts\pipeline
+
+    # The target queues to run the tests on.
+    # Currently runs on queues owned by the WinUI/XAML team. This should run on queues hosted by ProjectReunion in the future.
+    helixTargetQueues: 'Windows.10.Amd64.ClientRS5.Open.Xaml'
```

Comment:
Can you set things to run on non-Open queues when running on an internal Pipeline. See [here](https://github.com/microsoft/microsoft-ui-xaml/blob/090fd4d11be744473324056d8d00dce797cdda6c/build/AzurePipelinesTemplates/MUX-RunHelixTests-Job.yml#L147) for an example.

Reply:
Thanks for the example!
Are there closed (non-Open) Helix queues that we could (or should) use for ProjectReunion?
ie: Did you want me to update this to use non-Open queues? (I'm not sure if this was an ask, or if you were pointing out how to use both types of queues...)

Reply:
When running outside of dev.azure.com/ms, use the closed queues, which have the same name, just without the .Open part of the name.

Reply:
Thanks! Will update in a moment.

Reply:
This looks good, do we want to target more versions than just RS5?

Reply:
Good point. I thought we could start with RS5 for now (since that is the lowest version supported), and add more later based on need. I also didn't want to slow down the PR builds too much. Perhaps we can try experimenting with adding other versions and see what the impact is to the time for builds.